README.rst

```diff
@@ -380,6 +380,17 @@ Once you have changes in the requirements-python.txt or in Hydra Dockerfile
 - change the version in docker/env/version
 - run ``./docker/env/build_n_push.sh`` to build and push to Docker Hub
 
+SCT test profiling
+===========================
+
+- set environment variable "SCT_PROFILE" to 1, or add "profile: 1" into yaml file
```

Comment:
It's better to use more descriptive name for this thing. What about `SCT_ENABLE_TEST_PROFILING` ? And make this variable boolean instead of int. We support it.